ImageLoader/Package.swift

```diff
@@ -1,7 +1,13 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 6.2
 
 import PackageDescription
 
+let swiftSettings: [SwiftSetting] = [
+	.swiftLanguageMode(.v6),
+	.defaultIsolation(.none),
```

Comment:
Since this module is a non-UI tool, we explicitly set the isolation to none, so in case Apple change default values in future, we are ready.